reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -390,31 +478,34 @@ class StaticHtmlReporter : Reporter {
         }
 
         project.vcsProcessed.let { vcsInfo ->
-            h3(excludedClass) { +"VCS Information" }
-
-            table("ort-report-metadata $excludedClass") {
-                tbody {
-                    tr {
-                        td { +"Type" }
-                        td { +vcsInfo.type.toString() }
-                    }
-                    tr {
-                        td { +"URL" }
-                        td { +vcsInfo.url }
-                    }
-                    tr {
-                        td { +"Path" }
-                        td { +vcsInfo.path }
-                    }
-                    tr {
-                        td { +"Revision" }
-                        td { +vcsInfo.revision }
+            details {
+                unsafe { +"<summary><strong id=\"$excludedClass\">VCS Information</strong></summary>" }
```

Comment:
Is there no way to use the `kotlinx.html` DSL here?

Reply:
I found it elsewhere in this file and used it on the basis if the original author could, and he had more experience with Kotlin than I, I should be able to as well. If it's a really bad practice, perhaps we need a separate PR?

Reply:
Indeed the `summary` tag was [added](https://github.com/Kotlin/kotlinx.html/issues/127) to *kotlinx.html*, but the required version 0.7.2 was not released yet.